provision.sh

```diff
@@ -105,6 +105,18 @@ curl -OL https://github.com/openshift/origin/releases/download/v3.11.0/openshift
 tar -zxf openshift-origin-client-tools-v3.11.0-0cbc58b-linux-64bit.tar.gz
 sudo mv $HOME/openshift-origin-client-tools-v3.11.0-0cbc58b-linux-64bit/oc /usr/local/bin
 
+echo "Installing kubectl binary"
+sudo bash -c 'cat >> /etc/yum.repos.d/kubernetes.repo' << EOF
+[kubernetes]
+name=Kubernetes
+baseurl=https://packages.cloud.google.com/yum/repos/kubernetes-el7-x86_64
+enabled=1
+gpgcheck=1
+repo_gpgcheck=1
+gpgkey=https://packages.cloud.google.com/yum/doc/yum-key.gpg https://packages.cloud.google.com/yum/doc/rpm-package-key.gpg
+EOF
+sudo yum install -y kubectl
```

Comment:
You shouldn't do this.  You should copy `oc` to `kubectl`

Reply:
oh! didn't think of that